backend/typescript/models/entity.pgmodel.ts

```diff
@@ -14,7 +14,7 @@ export default class Entity extends Model {
   enum_field!: Letters;
 
   @Column({ type: DataType.ARRAY(DataType.STRING) })
-  string_array_field!: string[];
+  string_array_field!: [string];
```

Comment:
Sorry just saw your note on this:
> string[] was replaced with [string] in the backend because after the mongoose update the backend threw errors

I can help take a look, we should be finding a way to keep the types though.